apigenerator/resources/1 - Project/1 - BaseProject/Project/src/e_Infra/h_MCP/LlmServiceFactory.py

```
@@ -79,9 +75,9 @@ def get_llm_service(x_provider_header: str = None) -> LlmServiceBase:
         temperature_str = provider_runtime_settings.get("temperature") # Stored as string/float in JSON potentially
 
         if not model_name: # Fallback to environment default model name
-            if chosen_provider_name == "gemini": model_name = ENV_DEFAULT_GEMINI_MODEL_NAME
-            elif chosen_provider_name == "openai": model_name = ENV_DEFAULT_OPENAI_MODEL_NAME
-            elif chosen_provider_name == "anthropic": model_name = ENV_DEFAULT_ANTHROPIC_MODEL_NAME
+            if chosen_provider_name == "gemini": model_name = os.environ.get('ENV_DEFAULT_GEMINI_MODEL_NAME')
```

Ruff (0.12.2)

Multiple statements on one line (colon)

(E701)